public/js/utils/redux/middleware/log.js

```diff
@@ -9,7 +9,7 @@
  */
 function log({ dispatch, getState }) {
   return next => action => {
-    console.log("[DISPATCH]", JSON.stringify(action, null, 2));
+    console.log(`[DISPATCH ${action.type}]`, action);
```

Comment:
The other thing is try/CI: printing the full object always made the logs from there useful, instead of things like `{ prop: [Object] }`